poll/public/html/poll.html

```diff
@@ -1,42 +1,46 @@
 {{ js_template|safe }}
 <div class="poll-block themed-xblock" data-private="{% if private_results %}1{% endif %}" data-can-vote="{% if can_vote %}1{% endif %}">
-  <div class="poll-block-form-wrapper">
-    <h3 class="poll-header">{{display_name}}</h3>
-    <form>
-        <div class="poll-question-container">
-            {{question|safe}}
-        </div>
-        <ul class="poll-answers">
-        {% for key, value in answers %}
-            <li class="poll-answer">
-                <input type="radio" name="choice" id="{{url_name}}-answer-{{key}}" value="{{key}}" {% if choice == key %}checked{% endif %}/>
-                {% if value.img %}
-                <div class="poll-image">
-                    <label for="{{url_name}}-answer-{{key}}" class="poll-image-label">
-                        <img src="{{value.img}}" alt="{{value.img_alt|default_if_none:''}}"/>
-                    </label>
-                </div>
-                {% endif %}
-                <label class="poll-answer" for="{{url_name}}-answer-{{key}}">{{value.label|safe}}</label>
-            </li>
-        {% endfor %}
-        </ul>
-        <input class="input-main" type="button" name="poll-submit" value="{% if choice %}Resubmit{% else %}Submit{% endif %}" disabled />
-    </form>
-    <div class="poll-voting-thanks{% if not choice or can_vote %} poll-hidden{% endif %}"><span>Thank you for your submission!</span></div>
-    <div class="poll-submissions-count poll-hidden">
-        You have used <span class="poll-current-count">{{ submissions_count }}</span>
-        out of <span class="poll-max-submissions">{{ max_submissions }}</span> submissions.
-    </div>
-    {% if feedback %}
-    <div class="poll-feedback-container{% if not choice %} poll-hidden{% endif %}">
-        <hr />
-        <h3 class="poll-header">Feedback</h3>
-        <div class="poll-feedback">
-            {{feedback|safe}}
-        </div>
-    </div>
-    {% endif %}
+    <div class="poll-block-form-wrapper">
+      <h3 class="poll-header">{{display_name}}</h3>
+      <form>
+          <div class="poll-question-container">
+              {{question|safe}}
+          </div>
+          <ul class="poll-answers">
+          {% for key, value in answers %}
+              <li class="poll-answer">
+                  <div class="poll-input-container">
+                      <input type="radio" name="choice" id="{{url_name}}-answer-{{key}}" value="{{key}}" {% if choice == key %}checked{% endif %}/>
```

Comment:
@Kelketek Nit: You might want to reformat this to span multiple lines; it's a bit hard to read right now. Same goes for L28 and L30 below.